src/DataDefinitionsBundle/Form/Type/ExportProvider/CsvProviderType.php

```diff
@@ -26,6 +26,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder
             ->add('delimiter', TextType::class)
-            ->add('enclosure', TextType::class);
+            ->add('enclosure', TextType::class)
+            ->add('escape', TextType::class, ['empty_data' => '']);
```

Comment:
The issue here is we want to keep the original behaviour by default, but allow setting to empty. Do you agree @dpfaffenbauer?

Reply:
Never mind, we have the default set in JS and the code will optionally set it here. If you save the form, it will default to `\\` in JS. If you haven't saved the form, it will not have the value defined, in which case the interpeter will not use it.